tests/unit_tests/tests_launch/test_launch.py

```diff
@@ -0,0 +1,36 @@
+import os
+import time
+
+import wandb
+from wandb.sdk.internal.internal_api import Api as InternalApi
+from wandb.sdk.launch.launch_add import launch_add
+
+
+def test_launch_delete_queued_run(relay_server, runner, user, monkeypatch):
+    queue = "default"
+    proj = "test"
+    uri = "https://github.com/gtarpenning/wandb-launch-test"
+
+    api = InternalApi()
+    os.environ["WANDB_PROJECT"] = proj  # required for artifact query
+
+    # create project
+    run = wandb.init(project=proj)
```

Comment:
There absolutely must be a better way of creating a project for testing than this....